airflow/www/utils.py

```diff
@@ -132,6 +135,7 @@ def get_mapped_summary(parent_instance, task_instances):
         "end_date": group_end_date,
         "mapped_states": mapped_states,
         "try_number": get_try_count(parent_instance._try_number, parent_instance.state),
+        "execution_date": parent_instance.execution_date,
```

Comment:
Shouldn't this be logical_date ?

Reply:
`execution_date` is the term used in the HTML template code where its referenced by graph view. Changing this to logical_date will require changes to frontend references and increase the scope of the PR.

https://github.com/apache/airflow/commit/e16207409998b38b91c1f1697557d5c229ed32d1